js-exercises/update-inventory/updateInventory.js

```diff
@@ -0,0 +1,26 @@
+function updateInventory(...args) {
```

Comment:
when we know only two arguments are to be passed, so instead of using ...args directly currentInventoryArray, freshDeliveryArray could have been used